### PR TITLE
[documentation] Adjusted documentation to make it more user friendly

### PR DESCRIPTION
**Pull request by ondrejkouril**
#### Description
Adjusted documentation to make it more user friendly

**Tracking Issue:** https://swicloud.atlassian.net/browse/NH-97805

#### Testing
User review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,64 @@
 # SolarWinds OpenTelemetry Collector
-Distribution of OpenTelemetry Collector with all available components
-bundled within from [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector/tree/main)
-and [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+SolarWinds OpenTelemetry Collector is a distribution of OpenTelemetry Collector with components
+bundled from [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector/tree/main)
+and [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib). It also contains specific SolarWinds components for easier usage and enhanced metric collection.
```
Comment (simek-m) [inline README.md:4]:
```suggestion
and [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib). It also contains specific SolarWinds components for easier usage and enhanced telemetry collection.
```
? 

Logs and traces are supported too.

File: build/docker/README.md [hunk 2]
```diff
@@ -4,8 +4,12 @@ Docker image for _SolarWinds OpenTelemetry Collector_:
  - runs set of tests validating correct functionality of components used
  - starts the Collector at entrypoint, expecting input config located by default as `/opt/default-config.yaml`
 
-## How to Build
-To build the image, simply run docker build command, i.e.
+## Obtain an Image
```
Comment (simek-m) [inline build/docker/README.md:7]:
```suggestion
## Obtaining the Image
```
`Getting the Image`

File: README.md [hunk 3]
```diff
@@ -1,7 +1,64 @@
...
+
+After successful build, `solarwinds-otel-collector` should be present in `cmd/solarwinds-otel-collector`.
+
+Run `solarwinds-otel-collector --config=config.yaml`.
+
+## Getting Started
+Configuration for SolarWinds OTel Collector has to contain [SolarWinds Extension](./extension/solarwindsextension/README.md) and [Solarwinds Exporter](./exporter/solarwindsexporter/README.md). 
+
+### Example with docker
+1. Generate your ingestion in SWO. See [API Tokens](https://documentation.solarwinds.com/en/success_center/observability/content/settings/api-tokens.htm).
```
Comment (simek-m) [inline README.md:31]:
```suggestion
1. Generate your ingestion token in SWO. See [API Tokens](https://documentation.solarwinds.com/en/success_center/observability/content/settings/api-tokens.htm).
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,64 @@
...
+2. Create a `config.yaml` file that contains configuration for the SolarWinds OTel Collector. Insert the ingestion token and choose correct data center (na-01, na-02, na-03). Specify the collector name.
```
Comment (simek-m) [inline README.md:32]:
```suggestion
2. Create a `config.yaml` file that contains configuration for the SolarWinds OTel Collector. Insert the ingestion token and choose a correct data center (na-01, na-02, na-03). Specify the collector name.
```

Comment (adschr) [inline README.md:32]:
```suggestion
2. Create a `config.yaml` file that contains configuration for the SolarWinds OTel Collector. Insert the ingestion token and choose correct data center (na-01, na-02, eu-01). Specify the collector name.
```

**Review by simek-m — APPROVED** [review]

**Review by adschr — APPROVED** [review]

**Pull request merged**